systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -127,7 +127,7 @@ public static void setClassResources() {
                             resource.getKind(), resource.getMetadata().getName(), resource.getMetadata().getNamespace());
                     operation.inNamespace(resource.getMetadata().getNamespace())
                             .withName(resource.getMetadata().getName())
-                            .withPropagationPolicy(DeletionPropagation.FOREGROUND)
+                            .withPropagationPolicy(DeletionPropagation.BACKGROUND)
```

Comment:
Why did these have to change?

Reply:
Honestly, I don't know. Nobody was able to tell me why this was used and what was the expectation. What it was doing was that it set a finalizer but did not wait for anything. And the finalizer blocked the deletion of the CRDs later.